desktop_version/src/Graphics.h

```diff
@@ -134,17 +146,63 @@ class Graphics
+    void draw_grid_tile(SDL_Texture* texture, int t, int x, int y, int width, int height);
+    void draw_grid_tile(SDL_Texture* texture, int t, int x, int y, int width, int height, int r, int g, int b, int a, int scalex, int scaley);
+    void draw_grid_tile(SDL_Texture* texture, int t, int x, int y, int width, int height, int r, int g, int b, int a);
+    void draw_grid_tile(SDL_Texture* texture, int t, int x, int y, int width, int height, int r, int g, int b, int scalex, int scaley);
+    void draw_grid_tile(SDL_Texture* texture, int t, int x, int y, int width, int height, int r, int g, int b);
+    void draw_grid_tile(SDL_Texture* texture, int t, int x, int y, int width, int height, SDL_Color color, int scalex, int scaley);
+    void draw_grid_tile(SDL_Texture* texture, int t, int x, int y, int width, int height, SDL_Color color);
+
     void updatetextboxes(void);
     void drawgui(void);
 
-    void drawsprite(int x, int y, int t, int r, int g, int b);
-    void drawsprite(int x, int y, int t, SDL_Color color);
+    void draw_sprite(int x, int y, int t, int r, int g, int b);
+    void draw_sprite(int x, int y, int t, SDL_Color color);
+
+    void scroll_texture(SDL_Texture* texture, int x, int y);
 
     void printcrewname(int x, int y, int t);
     void printcrewnamedark(int x, int y, int t);
 
     void printcrewnamestatus(int x, int y, int t, bool rescued);
 
+    int set_render_target(SDL_Texture* texture);
+
+    int set_texture_color_mod(SDL_Texture* texture, Uint8 r, Uint8 g, Uint8 b);
+
+    int set_texture_alpha_mod(SDL_Texture* texture, Uint8 alpha);
+
+    int query_texture(SDL_Texture* texture, Uint32* format, int* access, int* w, int* h);
+
+    int set_blendmode(SDL_BlendMode blendmode);
+    int set_blendmode(SDL_Texture* texture, SDL_BlendMode blendmode);
+
+    int clear(int r, int g, int b, int a);
+    int clear();
+
+    int copy_texture(SDL_Texture* texture, SDL_Rect* src, SDL_Rect* dest);
+    int copy_texture(SDL_Texture* texture, SDL_Rect* src, SDL_Rect* dest, double angle, SDL_Point* center, SDL_RendererFlip flip);
+
+    int set_color(int r, int g, int b, int a);
+    int set_color(int r, int g, int b);
+    int set_color(SDL_Color color);
+
+    int fill_rect(SDL_Rect* rect, const int r, const int g, const int b, const int a);
+    int fill_rect(const int x, const int y, const int w, const int h, const int r, const int g, const int b, const int a);
+    int fill_rect(const int x, const int y, const int w, const int h, const int r, const int g, const int b);
+    int fill_rect(const int r, const int g, const int b, const int a);
+    int fill_rect(SDL_Rect* rect, const int r, const int g, const int b);
+    int fill_rect(const int r, const int g, const int b);
+    int fill_rect(SDL_Rect* rect, const SDL_Color color);
+    int fill_rect(const int x, const int y, const int w, const int h, const SDL_Color color);
+    int fill_rect(const SDL_Color color);
```

Comment:
In header files, `const` does not matter for non-pointer arguments.
```suggestion
    int fill_rect(const SDL_Rect* rect, int r, int g, int b, int a);
    int fill_rect(int x, int y, int w, int h, int r, int g, int b, int a);
    int fill_rect(int x, int y, int w, int h, int r, int g, int b);
    int fill_rect(int r, int g, int b, int a);
    int fill_rect(const SDL_Rect* rect, int r, int g, int b);
    int fill_rect(int r, int g, int b);
    int fill_rect(const SDL_Rect* rect, SDL_Color color);
    int fill_rect(int x, int y, int w, int h, SDL_Color color);
    int fill_rect(SDL_Color color);
```